Make organizers and meetings arrays

segiddins opened the pull request:
Closes #6.

@apontious you'll need to update the rest of the yaml files.

The design for chapter pages will probably also need some tweaking now.

segiddins committed:
Make organizers and meetings arrays

apontious committed:
Tweaks to chapter format and chapter page appearance.

apontious committed:
First pass at updating all chapters to new format.

Including adding in multiple organizers and reorganizing chapters with multiple meetings into proper slots. Plus some corrections.

NSCoderNight commented:
OK, chapter information updated.

I still need to update README for GitHub repository and give the chapters another once-over.

apontious committed:
Added San Diego chapter information in new format.

apontious committed:
Tweak meeting web page.

Added more text to what's shown for each chapter, so that the chapters themselves don't need as much text (and can in theory be used in different formats in the future). Specifically, now there are "Location:", "Day and Frequency:", and "Time:" labels, so the actual chapter text doesn't need to say "Location varies", it can just say "Varies" for the location field.
